usaspending_api/recipient/management/sql/recipient_lookup/100_finalize.sql

```diff
@@ -103,7 +103,7 @@ ON CONFLICT(recipient_hash) DO NOTHING;
 
 DO $$ BEGIN RAISE NOTICE 'Deleting duplicate DUNS records that are superceded by records with UEI populated.'; END $$;
 
-DELETE FROM public.recipient_lookup WHERE recipient_hash IN (
+DELETE FROM rpt.recipient_lookup WHERE recipient_hash IN (
     SELECT rl.recipient_hash
     FROM recipient_lookup rl
```

Comment:
Do you need to include the schema here?

Reply:
Decided to not change places where the schema was previously used. Realistically the schema name is not required here.